cypress/fixtures/Test_chat_partial_execution.json

```diff
@@ -0,0 +1,127 @@
+{
+  "nodes": [
+    {
+      "parameters": {
+        "assignments": {
+          "assignments": [
+            {
+              "id": "0c345346-8cef-415c-aa1a-3d3941bb4035",
+              "name": "text",
+              "value": "=Set 1 with chatInput: {{ $json.chatInput }}",
```

Comment:
Thank you for updating test data, now it's clear that the chat message propagates correctly.